weaviate/collections/classes/config.py

```diff
@@ -996,6 +996,39 @@ class _CollectionConfigUpdate(_ConfigUpdateModel):
         default=None, alias="multi_tenancy_config"
     )
 
+    def __check_quantizers(
+        self,
+        quantizer: Optional[_QuantizerConfigUpdate],
+        vector_index_config: dict,
+    ) -> None:
+        if (
+            (
+                isinstance(quantizer, _PQConfigUpdate)
+                and (
+                    vector_index_config["bq"]["enabled"]
+                    or vector_index_config.get("sq", {"enabled": False})["enabled"]
+                )
+                is True
```

Comment:
why the `is True`?

Reply:
Good point, will remove. I wonder why all the linting and static checking didn't catch that 🤔